ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -210,7 +210,7 @@ Get Model Route Via UI
     [Arguments]    ${model_name}
     # TODO: Open model serving home page if needed?
     # Click on Inference Endpoints link
-    ${endpoint_link}=    Set Variable    //td[@data-label='Name']//div[@class and text()='${model_name}']//ancestor::tr//td[@data-label='Inference endpoint']//button
+    ${endpoint_link}=    Set Variable    //a[@data-testid="metrics-link-test-model" and text()="${model_name}"]/ancestor::tr//td//button[@data-testid="internal-external-service-button"]
```

Comment:
Are you sure that this works on 2.15? I checked in the upgrade cluster, and there was no `@data-testid="metrics-link-test-model"`

Reply:
@FedeAlonso Fixed it

Reply:
@Raghul-M I mean, are you sure that the data-testid exists in 2.15? I modified that path yesterday (here https://github.com/red-hat-data-services/ods-ci/pull/2057 )  because I wasn't able to find any data-testid label in 2.15 (I used the upgrade cluster)

Reply:
@FedeAlonso  Yes, Fede, it's not available in 2.15. I have removed it and made the necessary changes.

Reply:
Ok, I see something a little bit weird. 
The modifications you are adding are going to be conflicting with the modifications I added two days ago. Most of them are the same, but others are not. 
Could you try to rebase or update your branch with releases/2.15 and fix the conflicts if it's needed?

Reply:
@FedeAlonso OKay let me check and rework on it